cmd/package_test.go

```diff
@@ -315,12 +315,11 @@ func (s *suite) TestDescribePackage(c *C) {
 
 func (s *suite) TestRecursiveRunYamls(c *C) {
 	// Prepare.
-	s.importFakeOSvBootstrapPkg(c)
 	s.importFakeDemoPkg(c)
 	s.requireFakeDemoPkg(c)
 
 	// This is what we're testing here.
-	err := CollectPackage(s.repo, s.packageDir, false, "", false)
+	err := CollectPackage(s.repo, s.packageDir, false, "", []string{}, false)
```

Comment:
Why have this been changed to have no _implicit_ requirement? I think I am misunderstanding something :(